doc/concepts/promotion.rst

```diff
@@ -1,6 +1,20 @@
+=====================
 Diffing and Promotion
 =====================
 
+Dune writes its build output to a separate directory from your source code tree 
+(usually ``_build/<profile_name>``). You can use Diffing and Promotion flows 
```

Comment:
It's not a profile but context. I wouldn't go into specifics beyond `_build` really.